reference/conanfile/methods.rst

```diff
@@ -107,7 +107,7 @@ The syntax of copy is as follows:
 
 .. code-block:: python
 
-   self.copy(pattern, dst, src, keep_path=False, symlinks=None, excludes=None)
+   self.copy(pattern, dst, src, keep_path=True, links=False, symlinks=None, excludes=None, ignore_case=False)
```

Comment:
Yes, that is right, there was a mistake in the docs, keep_path default is True. But better not show ``links`` it might eventually be removed. Thanks!